src/helpers/urlUtils.ts

```diff
@@ -66,10 +66,6 @@ export const stripNonAscii = (s: string): string => {
 };
 
 export const getLinkFromText = (text: string): string | undefined => {
-    const httpLink = text.match(/(http.?:\/\/.*?)( |$)/);
-    if (httpLink != null) {
-        return httpLink[1];
-    }
     const bzzFeedLink = text.match(/(bzz-feed:\/\?user=0x[a-f0-9]{40})( |$)/);
```

Comment:
It would be better to have a separate function for explicitly extract `bzz-feed` links from the text and that function could be called from here as well.